openupgradelib/openupgrade_merge_records.py

```diff
@@ -334,7 +334,7 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
             if op == 'merge':
                 if not getattr(target_record, field.name) and l and not \
                         vals.get(field.name):
-                    vals[field.name] = l[:1]
+                    vals[field.name] = [x for x in l if x][:1]
```

Comment:
You should filter this before here, for having proper comparison on `if` sentence.

Reply:
done